terraform/environments/development-eu-west-2/devops1/vars

```diff
@@ -0,0 +1,20 @@
+aws_bucket = "development-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-development-terraform-state-london"
+aws_region = "eu-west-2"
+environment = "devops1"
+deploy_to = "development"
+state_prefix = "env:/development"
+workspace_key_prefix = "test-data-generator-stack"
```

Comment:
Can this be specified via a default instead?

Reply:
Done. Moved workspace_key_prefix assignment from vars to root module variables.tf

Reply:
These look to still be here

Reply:
As per offline chat, seems these need to be here for the terraform script in Concourse resources.